.env.example

```
@@ -1,2 +1,5 @@
 # Required for automated integration testing
 ZOWE_TEST_PROFILE=''
+ZOS_WORKFLOWS_SYSTEM=SYST
+ZOS_WORKFLOWS_OWNER=MYUSER
+ZOS_WORKFLOWS_SAF_ID=MYUSER
```

For consistency, can we either remove the quotes above, or add quotes on the new ones?

```suggestion
ZOWE_TEST_PROFILE=zosmf
ZOS_WORKFLOWS_SYSTEM=SYST
ZOS_WORKFLOWS_OWNER=MYUSER
ZOS_WORKFLOWS_SAF_ID=MYUSER
```